src/App.css

```diff
@@ -52,4 +52,47 @@ ul {
   padding: 20px;
   max-width: 600px;
   text-align: center;
+  max-height: auto;
+}
+
+.comment-form {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: center;
+  margin-bottom: 20px;
+  padding: 30px;
+  max-width: 500px;
+  margin: 0 auto;
+}
+
+
+.comment-form textarea {
+  padding: 10px;
+  border: 1px solid #ccc;
+  border-radius: 5px;
+  margin-bottom: 10px;
+  width: 100%;
+  box-sizing: border-box;
+  height: 100px;
+  resize: vertical;
+}
+
+.comment-form label {
+  font-weight: bold;
+  margin-bottom: 5px;
+}
+
+.comment-form button {
+  padding: 10px 20px;
+  background-color: #007bff;
+  color: #fff;
+  border: none;
+  border-radius: 5px;
+  cursor: pointer;
+  max-width: 100px;
+}
+
+.comment-form button:hover {
+  background-color: #0056b3;
```

Comment:
When we post a very long line in the comments, for example a long URL, it doesn't wrap into that comment box and hangs out to the side. This needs to be fixed. 

See my comments left for this article: The Notorious MSG’s Unlikely Formula For Success